bundles/org.openhab.binding.lgwebos/src/main/java/org/openhab/binding/lgwebos/internal/MediaControlPlayer.java

```diff
@@ -42,7 +45,8 @@ public void onReceiveCommand(String channelId, LGWebOSHandler handler, Command c
         } else if (RewindFastforwardType.REWIND == command) {
             handler.getSocket().rewind(getDefaultResponseListener());
         } else {
-            logger.warn("Only accept PlayPauseType, RewindFastforwardType. Type was {}.", command.getClass());
+            logger.warn("Only accept PlayPauseType, RewindFastforwardType, RefreshType. Type was {}.",
```

Comment:
WARN level is really too high (look at the OH recommendations).
Should be DEBUG, max INFO.